velox/core/SimpleFunctionMetadata.h

```diff
@@ -364,8 +384,11 @@ class UDFHolder final
       const typename Exec::template resolver<TArgs>::in_type&... args) {
     if constexpr (udf_has_call) {
       return instance_.call(out, args...);
-    } else {
+    } else if constexpr (udf_has_callNullable) {
       return instance_.callNullable(out, (&args)...);
+    } else {
+      VELOX_UNREACHABLE(
```

Comment:
the code does not check callNullFree?
should the message be neither call nor callNullable are implemented?
or is there implicit dependency.